ui/app/components/ui/alert-message/alert-message.component.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import BoxedMessage from '../boxed-message'
+
+import success from '../../../../../app/images/icons/green-circle-check.svg'
+import danger from '../../../../../app/images/icons/red-triangle-exclaim.svg'
+import info from '../../../../../app/images/icons/blue-circle-info.svg'
+import warning from '../../../../../app/images/icons/yellow-bell.svg'
+
+const typeConfig = {
+  success: {
+    boxClass: 'alert-message--success',
+    iconSource: success,
+  },
+  danger: {
+    boxClass: 'alert-message--danger',
+    iconSource: danger,
+  },
+  info: {
+    boxClass: 'alert-message--info',
+    iconSource: info,
+  },
+  warning: {
+    boxClass: 'alert-message--warning',
+    iconSource: warning,
+  },
+}
+
+export default class AlertMessage extends Component {
+  static propTypes = {
+    type: PropTypes.oneOf(Object.keys(typeConfig)).isRequired,
+    children: PropTypes.node,
+  }
+
+  render () {
+    return (
+      <BoxedMessage
+        { ...typeConfig[this.props.type] }
+        message={this.props.children}
```

Comment:
This seems.... unconventional. `children` is usually expected to be a `node`, yet it's being passed in here as a prop expected to be a string.

Better to use either a string or a node in both places

Reply:
Fixed in 89058fe08